client/my-sites/people/invite-people/index.jsx

```diff
@@ -69,7 +70,10 @@ export default React.createClass( {
 			return value;
 		} );
 
-		this.setState( { usernamesOrEmails: filteredTokens } );
+		this.setState( {
+			usernamesOrEmails: filteredTokens,
+			errorTokenHover: ! includes( filteredTokens, errorTokenHover ) ? false : errorTokenHover
```

Comment:
I think we can write this like: `errorTokenHover: includes( filteredTokens, errorTokenHover ) && errorTokenHover`